README.md

```diff
@@ -125,7 +188,7 @@ Sets a variable to a specific value based on its ID.
 Note: Policy to create the variable must have been already loaded
 otherwise you will get a 404 error during invocation.
 
-#### `apply_policy_file(policy_name, policy_file)`
+#### `load_policy_file(policy_name, policy_file)`
 
 Applies a file-based YAML to a named policy. This method only
 supports additive changes. Result is a dictionary object constructed
```

Comment:
This method supports additive changes only.

Reply:
this is a name of a function :)